src/h5web/vis-packs/core/models.ts

```diff
@@ -46,11 +53,17 @@ export interface AxisConfig {
   flip?: boolean;
 }
 
-export type AxisScale = PickD3Scale<
-  ScaleType.Linear | ScaleType.Log | ScaleType.SymLog | ScaleType.Sqrt,
-  number,
-  number
+export type VisxScaleConfig = Omit<
+  PickScaleConfig<'linear' | 'log' | 'symlog', number>,
+  'type'
 >;
+export type GammaScaleConfig = Omit<PickScaleConfig<'pow', number>, 'type'>;
+
+export type AxisScale =
+  | ScaleLinear<number, number>
+  | ScaleLogarithmic<number, number>
+  | ScaleSymLog<number, number>
+  | ScalePower<number, number>;
```

Comment:
You're missing `ScaleSqrt` I think

Reply:
`ScaleSqrt` is a particular `ScalePower`. I could add it explicitly tho.